library/src/main/java/com/google/android/exoplayer/text/webvtt/WebvttCue.java

```diff
@@ -26,20 +26,33 @@
 
   public final long startTime;
   public final long endTime;
+  public final Alignment lineAlignment;
```

Comment:
Can we please leave this class alone for the purposes of this change? It's not clear that adding more things here is the right thing to do with the new parameters that you're parsing. It's quite likely they'll be used to modify the existing line/position/alignment values that get passed to the super class).